### PR TITLE
realtime cleverage: considerations of theoric time 

### PR DESCRIPTION
**Pull request by patochectp**

### CODE REVIEW

**Comment by TeXitoi** [comment]
OK for me.

**Comment by antoine-de** [comment]
I think it would be nice to add integration tests on this (to check weither the `is_real_time` is really used :wink: )

**Comment by patochectp** [comment]
done

File: source/jormungandr/tests/proxy_realtime_cleverage_integration_tests.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+@dataset({'multiple_schedules': {'proxy_conf': MOCKED_PROXY_CONF}})
+class TestCleverageSchedules(AbstractTestFixture):
+    """
+    integration tests for cleverage
+
+    """
+    query_template = 'stop_points/{sp}/stop_schedules?data_freshness=realtime&_current_datetime=20160102T0800'
+
+    def test_stop_schedule_with_realtime_only(self):
+        """
```
Comment (antoine-de) [inline source/jormungandr/tests/proxy_realtime_cleverage_integration_tests.py]: can you add some general comments on the test ? (even if the name is quite clear ?)

else you can remove the empty `""" """` :smile:

File: source/jormungandr/tests/proxy_realtime_cleverage_integration_tests.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            scs = get_not_null(response, 'stop_schedules')
+            assert len(scs) == 1
+            # 2016-01-02 08:17:00
+            assert _get_schedule(response, 'SP_1', 'code A') == [
+                {'rt': True, 'dt': '20160102T081717'},
+                {'rt': True, 'dt': '20160102T091717'}
+            ]
+
+    def test_stop_schedule_with_theoric_and_realtime(self):
+        """
```
Comment (antoine-de) [inline source/jormungandr/tests/proxy_realtime_cleverage_integration_tests.py]: same here

**Review by antoine-de — CHANGES_REQUESTED** [review]
thanks for the integration tests!

**Comment by patochectp** [comment]
done

**Review by antoine-de — APPROVED** [review]

**Pull request merged**